django-stubs/http/request.pyi

```diff
@@ -26,6 +26,14 @@ class HttpHeaders(CaseInsensitiveMapping[str]):
     def __init__(self, environ: Mapping[str, Any]) -> None: ...
     @classmethod
     def parse_header_name(cls, header: str) -> str | None: ...
+    @classmethod
+    def to_wsgi_name(cls, header: str) -> str: ...
+    @classmethod
+    def to_asgi_name(cls, header: str) -> str: ...
+    @classmethod
+    def to_wsgi_names(cls, headers: Mapping[str, Any]) -> dict[str, Any]: ...
+    @classmethod
+    def to_asgi_names(cls, headers: Mapping[str, Any]) -> dict[str, Any]: ...
```

Comment:
These are unrelated to `cached_property`. For clarity, I'd prefer unrelated changes as a separate PR.

Reply:
yes my bad, mixed up some branches in the beginning, I wanted to put this in another PR too but forgot to rebase, will be more careful next time!